app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+	def full_title(page_title = '')
```

Comment:
- Method viết cách đầu dòng 2 spaces thay vì 8 spaces
- Dùng " thay cho ' trên toàn bộ file này